### PR TITLE
Replacing links to bugs.sun.com

### PR DESCRIPTION
**Pull request by jglick**
@reviewbybees

### CODE REVIEW

File: core/src/main/java/hudson/ExtensionFinder.java [hunk 1]
```diff
@@ -175,7 +175,7 @@ public boolean isRefreshable() {
      * from here.
      *
      * <p>
-     * See http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=6459208 for how to force a class initialization.
+     * See https://bugs.openjdk.java.net/browse/JDK-4993813 for how to force a class initialization.
```
Comment (jglick) [inline core/src/main/java/hudson/ExtensionFinder.java]: Following duplicate link.

File: core/src/main/java/hudson/util/jna/InitializationErrorInvocationHandler.java [hunk 2]
```diff
@@ -12,7 +12,6 @@
  *
  * @author Kohsuke Kawaguchi
  * @since 1.487
- * @see <a href="http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=7199848">Related bug report against JDK</a>
```
Comment (jglick) [inline core/src/main/java/hudson/util/jna/InitializationErrorInvocationHandler.java]: Seems to have been deleted/hidden.

**Comment by oleg-nenashev** [comment]
:bee: and :+1:

**Comment by recena** [comment]
:bee: Thanks for doing this work

**Pull request merged**